### PR TITLE
Change README.md Azure RTOS -> Eclipse ThreadX

### PR DESCRIPTION
**Pull request by MatthewKhouzam**
## PR checklist

This is a documentation patch only. It does not affect the image here. 

https://github.com/eclipse-threadx/threadx/raw/master/docs/threadx-features.png 

- [ ] Updated function header with a short description and version number
- [ ] Added test case for bug fix or new feature
- [ ] Validated on real hardware

### CODE REVIEW

**Review by wickste — APPROVED** [review]
Thanks for the contribution Matthew! I will update the image and some of links in a separate change shortly.

File: README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 
 We also provide [getting started guide](https://github.com/eclipse-threadx/getting-started) and [samples](https://github.com/eclipse-threadx/samples) using hero development boards from semiconductors you can build and test with.
 
-See [Overview of Azure RTOS ThreadX](https://learn.microsoft.com/en-us/azure/rtos/threadx/overview-threadx) for the high-level overview, and all documentation and APIs can be found in: [Azure RTOS ThreadX documentation](https://learn.microsoft.com/en-us/azure/rtos/threadx/). 
+See [Overview of ThreadX](https://learn.microsoft.com/en-us/azure/rtos/threadx/overview-threadx) for the high-level overview, and all documentation and APIs can be found in: [ThreadX documentation](https://learn.microsoft.com/en-us/azure/rtos/threadx/).
```
Comment (jserv) [inline README.md:15]: This hyperlink would not be proper since it is referred to Azure RTOS ThreadX instead of Eclipse ThreadX.

Comment (wickste) [reply README.md:15]: @jserv this has already been addressed in the latest version of the readme.md

**Comment by wickste** [comment]
> So... anything else to do? Basically I cannot merge the PR, it's up to the committers. ;)

@MatthewKhouzam I have already updated all readmes, links, images, etc. in all repos in the ThreadX organization. Please take a look if you see anything missing.

**Comment by MatthewKhouzam** [comment]
LGTM, so I will abandon the patch then.

**Pull request closed without merging**